src/Illuminate/Support/Str.php

```diff
@@ -56,8 +56,8 @@ public static function ascii($value, $language = 'en')
     {
         $languageSpecific = static::languageSpecificCharsArray($language);
 
-        if (! is_null($languageSpecific)) {
```

Comment:
https://github.com/kalessil/phpinspectionsea/blob/master/docs/probable-bugs.md#null-pointer-exceptions-prevention

> is_object(), is_null() and similar is_*() functions calls are not analyzed, instead we recommend to rely on null identity and instanceof operators;

All 'is_null(...)' calls can be safely replaced with 'null === ...' constructs (or 'null !== ...' if the original construct was negated).

Reply:
The rest of the framework uses `is_null`

Reply:
@ntzm yes, but it will be more clear and correct to use `null !== ...`. Should I revert this changes?

Reply:
It should stay consistent throughout the framework

Reply:
@ntzm Rolled back